integration-tests/structured-logging/plugins/structured-plugin-errors/gatsby-node.js

```diff
@@ -6,11 +6,18 @@ exports.onPreInit = ({ reporter }) => {
       category: "SYSTEM",
       docsUrl: `https://www.gatsbyjs.org/docs/gatsby-cli/#new`,
     },
+    "12345": {
+      text: context => `Error text is ${context && context.someProp}`,
+      level: "ERROR",
+      category: "SYSTEM",
+      docsUrl: `https://www.gatsbyjs.com/docs/cheat-sheet/`,
+    }
   })
 
   reporter.info("setErrorMap")
 
   if (process.env.PANIC_IN_PLUGIN) {
+    reporter.error({ id: "structured-plugin-errors_12345", context: { someProp: `MORE ERROR!` } })
```

Comment:
I'm using `.error` here so that multiple errors can be checked (as after `panic` it'll kill the build)

Reply:
The "MORE ERROR!" part makes it look like a `console.log` statement rather than an error message. Could it contain anything more relevant, or otherwise, be omitted?

Reply:
This is solely for the [context of the error](https://github.com/gatsbyjs/gatsby/blob/fix-error-map-double-prefix/integration-tests/structured-logging/plugins/structured-plugin-errors/gatsby-node.js#L10) to see if is passed through (this is also tested in the test). In a failing state of the test the text would be `There was an error`.